core/src/main/groovy/org/grails/GuidesFetcher.groovy

```diff
@@ -5,11 +5,14 @@ import groovy.transform.CompileDynamic
 import groovy.transform.CompileStatic
 import org.grails.model.Guide
 
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+
 @CompileStatic
 class GuidesFetcher {
 
     public static final String GUIDES_JSON = 'https://raw.githubusercontent.com/grails/grails-guides/gh-pages/guides.json'
-
+    private static final DateFormat dateFormat = new SimpleDateFormat('dd MMM yyyy', Locale.US)
```

Comment:
SimpleDateFormat isn't threadsafe (see the Javadoc), so it's best to create a new instance each time